metar.ts

```diff
@@ -64,3 +68,15 @@ export default async (msg: Message) => {
 	}
 	await sendMetarMessage(metar, msg);
 };
+
+function calcDA(icao_id:string,temp:number,alt_set:number) { // Calculate density altitude for ZAB airfields
+    icao_id = icao_id.toUpperCase();
+    var elevation: {[key:string]:number} = {KAMA:3649,KCVS:4295,KROW:3671,KELP:3962,KBIF:3947,KHMN:4093,KABQ:5355,KAEG:5837,KSAF:6349,KSKX:7095,KFHU:4719,KTUS:2643,KDMA:2704,KRYN:2419,KPHX:1135,KIWA:1384,KFFZ:1394,KGYR:969,KLUF:1085,KGEU:1071,KDVT:1478,KSDL:1510,KGXF:883,KPRC:5045,KSEZ:4831,KFLG:7015,KINW:4941,KGUP:6472};
+    if(elevation.hasOwnProperty(icao_id)) {
```

Comment:
Never been a big fan of `Object.hasOwnProperty`, instead just preferring the truthy check `if(elevation[icao_id])`.

Reply:
Removed and replaced